buildsystem/codecompliance/authors.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
 # Copyright 2014-2016 the openage authors. See copying.md for legal info.
```

Comment:
you have to update the copyright year, the utf-8 coding is enabled by default as it's python3.